src/community/arcsde/src/test/java/org/geoserver/web/data/store/arcsde/ArcSDECoverageStoreEditPanelTest.java

```diff
@@ -158,12 +158,8 @@ public void testRefreshRasterTableListBadConnectionParams() {
                         tester.getComponentFromLastRenderedPage(base + "tableNamePanel");
 
         tableChooserPanel.setSessionFactory(
-                new ISessionPoolFactory() {
-
-                    public ISessionPool createPool(final ArcSDEConnectionConfig config)
-                            throws IOException {
-                        throw new IOException("can't connect for some reason");
-                    }
+                config -> {
+                    throw new IOException("can't connect for some reason");
```

Comment:
Probably... in the specific case, arcsde is about to go the way of the dodo (see discussion on gt-devel) so it will be a non issue.
So far this PR has like 10 minutes effort in it, just run Intellij and pushed the results for discussion, so easy to simply re-run it at a later stage.